File: atv-topico-5/topico-5-js/frequencia.js
const frequencia = (a, b, c = 0) => {
  if (b.length > c) {
    if (a == b[c]) {
      return 1 + frequencia(a, b, c + 1);
    } else {
      return frequencia(a, b, c + 1);
    }
  }
  return 0;
};

console.log(frequencia(3, [1, 2, 3, 4, 3]));

File: atv-topico-5/topico-5/frequencia.hs

frequencia c [] = 0
frequencia c (x:d) = 
                if(c == x)
                  then 1 + frequencia c d
                  else frequencia c d



main = do
    a <- readLn :: IO Int
    b <- readLn :: IO [Int]
    print $ frequencia a b